vendor/raygui/raygui.odin

```diff
@@ -0,0 +1,555 @@
+package raygui
+
+import c "core:c/libc"
+import "vendor:raylib"
+
+when ODIN_OS == .Windows {
+	@(extra_linker_flags="/NODEFAULTLIB:libcmt")
+	foreign import lib {
+		"windows/raygui.lib",
+	}
+} else when ODIN_OS == .Linux  {
+	foreign import lib { 
+		"linux/libraygui.a",
+		// "system:dl",
+		// "system:pthread",
+	}
+} else when ODIN_OS == .Darwin {
+	when ODIN_ARCH == .arm64 {
+		foreign import lib {
+			"macos-arm64/libraygui.a",
+			// "system:Cocoa.framework",
+			// "system:OpenGL.framework",
+			// "system:IOKit.framework",
+		}
+	} else {
+		foreign import lib {
+			"macos/libraygui.a",
+			// "system:Cocoa.framework",
+			// "system:OpenGL.framework",
+			// "system:IOKit.framework",
+		}
+	}
+} else {
+	foreign import lib "system:raygui"
+}
+
+VERSION :: "4.0"
+
+// Style property
+GuiStyleProp :: struct {
```

Comment:
As everything is prefixed with `Gui` in this package already, it might be better to remove the prefix entirely.

Reply:
It is a separate package. But I think it's better to keep it just so naming stay consistent with the original library.